docs/examples/contrib/sqlalchemy/plugins/tutorial/full_app_with_init_plugin.py

```diff
@@ -74,7 +74,7 @@ async def update_item(item_title: str, data: TodoItem, transaction: AsyncSession
     return todo_item
 
 
-db_config = SQLAlchemyAsyncConfig(connection_string="sqlite+aiosqlite:///todo.sqlite")
+db_config = SQLAlchemyAsyncConfig(connection_string="sqlite+aiosqlite:///todo.sqlite", create_all=True)
```

Comment:
If I understand the changes to advanced-alchemy correctly, we'd also need to add `metadata=Base.metadata` to the config.

Reply:
https://github.com/jolt-org/advanced-alchemy/blob/bc51751897ff6ca56e2a21ddc9897248cc6e62ef/advanced_alchemy/config/common.py#L120-L126

Reply:
It does run without specifying which metadata to use. However, one could add it specifically. I am not too familiar with the details of SQLAlchemy, so not 100% sure what's needed.

[Here](https://github.com/jolt-org/advanced-alchemy/blob/05ea56e1d7f985f09596fa8948f9171b2ad97f2d/advanced_alchemy/config/common.py#L211) the default metadata is specified as `orm_registry.metadata`. I am not sure what exactly that entails.

Reply:
It runs, but have you confirmed that the tables are being created? 

We aren't using the advanced-alchemy base classes in this example. We create our own `Base` by creating the `class Base(DeclarativeBase): ...` class which all of the models in the example inherit from.

That means that `advanced_alchemy.base.orm_registry.metadata` doesn't know anything about the tables represented by the models in the example. And this is why we need to tell the config object about _our_ metadata.

Reply:
Awsome, after this change we'll be right to merge! Thanks.

Reply:
Ok, added the code and committed your change in the wording, too. Had to fix a couple of formatting issues that didn't show up in my local pre-commit run. Should work now.